includes/specials/SpecialUserrights.php

```diff
@@ -45,15 +106,6 @@ public function userCanExecute( User $user ) {
 		return $this->userCanChangeRights( $user, false );
```

Comment:
You have converted method `userCanChangeRights` to static. Please make sure you adjust all the places where this is called to call it as static method.